### ISSUE TITLE
RGB images of validation IDs not available for indoor-search mode

### ISSUE THREAD

**Issue by sshkhr** [wuancong/SYSU-MM01#2]
Hello and thank you for the excellent dataset.
I noticed that the 99 validation IDs from #334 to #433 are not available in the cam1 and cam2 which contain the indoor RGB images.

Could you please explain how was validation for indoor search mode ? 
Did you use the same parameters as the all-search mode ? 

Thanks

**Comment by wuancong** [comment]
Hi,

The parameters are kept the same as the all-search mode.

The validation IDs can be also regarded as a part of the training set. You could split a different validation set if needed.

**Comment by sshkhr** [comment]
Thanks for clarifying.
